model/userModel.js

```diff
@@ -1,24 +1,38 @@
 const Mongoose = require('mongoose');
+const Validator = require('validator');
+
+const isDateValid = (value) => {
+    return value < Date.now();
+}
 
 const userSchema = Mongoose.Schema({
     companyID: {
         type: Mongoose.Types.ObjectId,
-        required: true
+        required: [true, 'Company ID must be specified']
     },
     name: {
         type: String,
-        required: true
+        required: [true, 'User name must be specified'],
+        trim: true
     },
     surname: {
         type: String,
-        required: true
+        required: [true, 'User surname must be specified'],
+        trim: true
     },
     email: {
         type: String,
-        required: true
+        required: [true, 'User email must be specified'],
+        match: [/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/, 'User email is not valid']
```

Comment:
No. Use this: https://www.npmjs.com/package/validator in validate method.

Reply:
Fuck lol, fixed